package.json

```diff
@@ -49,7 +49,10 @@
     "image-type": "^4.1.0",
     "jpeg-js": "^0.4.4",
     "median-quickselect": "^1.0.1",
+    "ml-array-variance": "^1.1.8",
```

Comment:
Please check if we don't have this method in ml-spectra-processing so that you don't need to add one more package.

Same is true for ml-array-xy-covariance

Reply:
https://github.com/mljs/spectra-processing/blob/0e0361bab81e8a12c700b4429e468f92182370ad/src/x/xVariance.ts#L12-L14 can not be used ?